From f4e2142089487b7f5c99da949d7bf8d038724df7 Mon Sep 17 00:00:00 2001
From: William Hubbs <w.d.hubbs@gmail.com>
Date: Tue, 1 May 2018 13:28:57 -0500
Subject: [PATCH] Add _POSIX_C_SOURCE definition to Linux build

We need this to allow builds on uclibc-ng based systems.

X-Gentoo-Bug: 650908
X-Gentoo-Bug-URL: https://bugs.gentoo.org/show_bug.cgi?id=650908
---
 mk/os-Linux.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/os-Linux.mk b/mk/os-Linux.mk
index 2479ff5b..9e8a1485 100644
--- a/mk/os-Linux.mk
+++ b/mk/os-Linux.mk
@@ -11,7 +11,7 @@
 SFX=		.Linux.in
 PKG_PREFIX?=	/usr
 
-CPPFLAGS+=	-D_DEFAULT_SOURCE
+CPPFLAGS+=	-D_DEFAULT_SOURCE -D_POSIX_C_SOURCE=200809L
 LIBDL=		-Wl,-Bdynamic -ldl
 
 ifeq (${MKSELINUX},yes)
-- 
2.16.1

